doghouse/server/token.go

```diff
@@ -0,0 +1,42 @@
+package server
+
+import (
+	"context"
+	"crypto/rand"
+	"fmt"
+	"io"
+
+	"github.com/haya14busa/reviewdog/doghouse/server/storage"
+)
+
+func GenerateRepositoryToken() string {
+	return securerandom(8)
+}
+
+func securerandom(n int) string {
+	b := make([]byte, n)
+	io.ReadFull(rand.Reader, b)
+	return fmt.Sprintf("%x", b)
+}
+
+func GetOrGenerateRepoToken(ctx context.Context, s storage.GitHubRepositoryTokenStore,
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function GetOrGenerateRepoToken should have comment or be unexported